pylint/checkers/classes.py

```diff
@@ -266,22 +267,56 @@ def _has_different_parameters_default_value(original, overridden):
     return False
 
 
-def _has_different_parameters(original, overridden, dummy_parameter_regex):
+def _has_different_parameters(
+    original: List[astroid.AssignName],
+    overridden: List[astroid.AssignName],
+    dummy_parameter_regex: Pattern,
+    counter: int,
+):
```

Comment:
Here it was a boolean before it look like it's now a list of string.
```suggestion
) -> List[str]:
```